### DISCUSSION TITLE
@types/vue3-google-map availability

### DISCUSSION THREAD

**Discussion by alessandrorubio** [inocan-group/vue3-google-map#237] [Q&A] [answered]
Currently no component is being found using typescript and composition API. For instance this is my code:
```
...
<script setup lang="ts">
  <GoogleMap api-key="YOUR_GOOGLE_MAPS_API_KEY">
  </GoogleMap>
</script>
...
```
Is this package usable for vue components using typescript?

**Comment by HusamElbashir** [comment, marked as answer]
Hi @alessandrorubio 

Yes it should be usable with typescript. If you're having issues make sure to include `@types/google.maps` in your `devDependencies`.
